modules/mobile/spec/support/vcr_cassettes/rx_refill/prescriptions/gets_a_list_of_all_prescriptions.yml

```diff
@@ -0,0 +1,280 @@
+             "prescriptionName":"ASPIRIN 325MG EC TAB",
+             "dispensedDate":null,
+             "stationNumber":"989"
+          },
+          {
+             "refillStatus":"hold",
+             "refillSubmitDate":null,
+             "refillDate":"Sun, 26 Jun 2016 00:00:00 EDT",
+             "refillRemaining":11,
+             "facilityName":"DAYT29",
+             "isRefillable":false,
+             "isTrackable":false,
+             "prescriptionId":13651311,
+             "orderedDate":"Sun, 26 Jun 2016 00:00:00 EDT",
+             "quantity":10,
+             "expirationDate":"Tue, 27 Jun 2017 00:00:00 EDT",
+             "prescriptionNumber":"2719559",
+             "prescriptionName":"DIGOXIN (LANOXIN) 0.25MG TAB",
+             "dispensedDate":null,
+             "stationNumber":"989"
+          },
+          {
+             "refillStatus":"expired",
+             "refillSubmitDate":null,
+             "refillDate":"Thu, 19 May 2016 00:00:00 EDT",
+             "refillRemaining":11,
+             "facilityName":"DAYT29",
+             "isRefillable":false,
+             "isTrackable":false,
+             "prescriptionId":13568746,
+             "orderedDate":"Thu, 19 May 2016 00:00:00 EDT",
+             "quantity":10,
+             "expirationDate":"Sat, 20 May 2017 00:00:00 EDT",
+             "prescriptionNumber":"2719535",
+             "prescriptionName":"TORSEMIDE 100MG TAB",
+             "dispensedDate":null,
+             "stationNumber":"989"
+          },
+          {
+             "refillStatus":"expired",
+             "refillSubmitDate":null,
+             "refillDate":"Fri, 29 Jan 2016 00:00:00 EST",
+             "refillRemaining":8,
+             "facilityName":"DAYT29",
+             "isRefillable":false,
+             "isTrackable":false,
+             "prescriptionId":13357873,
+             "orderedDate":"Fri, 29 Jan 2016 00:00:00 EST",
+             "quantity":10,
+             "expirationDate":"Sun, 29 Jan 2017 00:00:00 EST",
+             "prescriptionNumber":"2719471",
+             "prescriptionName":"ADEFOVIR DIPIVOXIL 10MG TAB",
+             "dispensedDate":null,
+             "stationNumber":"989"
+          },
+          {
+             "refillStatus":"discontinued",
+             "refillSubmitDate":null,
+             "refillDate":"Sun, 24 Jan 2016 00:00:00 EST",
+             "refillRemaining":11,
+             "facilityName":"DAYT29",
+             "isRefillable":false,
+             "isTrackable":false,
+             "prescriptionId":13357874,
+             "orderedDate":"Sun, 24 Jan 2016 00:00:00 EST",
+             "quantity":15,
+             "expirationDate":"Tue, 24 Jan 2017 00:00:00 EST",
+             "prescriptionNumber":"2719472",
+             "prescriptionName":"MIRTAZAPINE 15MG TAB",
+             "dispensedDate":null,
+             "stationNumber":"989"
+          }
+       ],
+       "failedStationList":"",
+       "lastUpdatedTime":"Tue, 17 Jan 2012 00:00:00 EST"
```

Comment:
Seems like only difference between existing VCRs and the example responses from  documentation is that the failed station list and last updated time are the last two fields instead of the first two, which functionally shouldn't have any difference but I changed it to keep it as close to reality as possible.